src/mm/state.c

```diff
@@ -150,6 +150,8 @@ bool LogState(unsigned int lid) {
 				LPS[lid]->from_last_ckpt = 0;
 			}
 			break;
+		case INCREMENTAL_STATE_SAVING:
+			take_snapshot = true;
 
 		default:
```

Comment:
This has missing break, so it executes rootsim_error.
It should have the same case as PERIODIc_STATE_SAVING.

see the following

```suggestion

		case INCREMENTAL_STATE_SAVING:
		case PERIODIC_STATE_SAVING:
			if(LPS[lid]->from_last_ckpt >= LPS[lid]->ckpt_period) {
				take_snapshot = true;
				LPS[lid]->from_last_ckpt = 0;
			}
			break;
		default:
```